### ISSUE TITLE
Threema Web: "This browser is not supported"

### ISSUE THREAD

**Issue by frommMoritz** [ramboxapp/community-edition#2076]
### Steps to reproduce

1. Open Threema Web in Rambox

### Actual behavior
"This browser is not supported

Please use the latest version of Google Chrome, Mozilla Firefox, Opera or Safari (iOS only), otherwise the web client might not work properly."

### ENVIRONMENT

> Rambox: v0.6.3

> OS: Linux Mint 19 Tara

> Arch: x86_64

**Comment by saenzramiro** [comment]
Thank you for reporting this. We will fix it in next version.

**Comment by My-Name-Is-Adam** [comment]
Don't bother with Rambox.. Developer has abandoned it for the paid version. 

Use Wavebox. Been using this for a while and it is so much better than Rambox. Even made a few suggestions on Github and they implemented them in the next release. 

Lots of big names use them such as Shopify, Spotify, Adobe, and Microsoft to name just a few. (at least according to the website)

https://wavebox.io

Check it out and let me know what you think :)

**Comment by RicoLaaa** [comment]
same applies to slack

**Comment by dbrgn** [comment]
Hi. The minimal Chromium version supported by Threema Web is 65, released almost a year ago (2018-03-14). See [`BROWSER_MIN_VERSIONS` in `app.ts`](https://github.com/threema-ch/threema-web/blob/master/src/app.ts).

This doesn't mean that Threema Web won't work anymore, but we won't add patches for things that are supported in 65+ but not in older versions. Furthermore, from a security perspective a current version of Chromium should be used.

TLDR: Threema Web can still be used by clicking the "Continue anyway" button, but Rambox should update to an Electron version that uses Chromium 65 or higher.

**Comment by fvulich** [comment]
The new upcoming version 0.6.4 it's gonna update chromium and this issue will be fixed
